.github/workflows/ci.yaml

```diff
@@ -13,23 +13,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8","3.9", "3.10", "3.11"]
+        python-version: ["3.8","3.9", "3.10", "3.11", "3.12", "3.13"]
         experimental: [false]
         # include:
-        #   - python-version: "3.12"
+        #   - python-version: "3.14"
         #     experimental: true
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v6
+        with:
+          enable-cache: true
 
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version-file: ".python-version"
```

Comment:
if you want to test in multiple python versions, you have to use the matrix reference